tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -504,3 +511,35 @@ setup_file() {
     rm -rf $FLUVIO_HOME_DIR
     assert_success
 }
+
+@test "Sets the desired version after installing it" {
+    run bash -c '$FVM_BIN self install'
+    assert_success
+
+    # Sets `fvm` in the PATH using the "env" file included in the installation
+    source ~/.fvm/env
+
+    run bash -c 'fvm current'
+    assert_line --index 0 "warn: No active version set"
+    assert_line --index 1 "help: You can use fvm switch to set the active version"
+    assert_success
+
+    run bash -c 'fvm install 0.10.14'
```

Comment:
Can you put version as variable?

Reply:
Fixed here: [a1d277f](https://github.com/infinyon/fluvio/pull/3604/commits/a1d277f6cd3252361e6b6de0ac9ce4d4705bbef5)